rpc/src/parsed_token_accounts.rs

```diff
@@ -81,22 +91,37 @@ where
 
 /// Analyze a mint Pubkey that may be the native_mint and get the mint-account owner (token
 /// program_id) and decimals
-pub fn get_mint_owner_and_decimals(bank: &Bank, mint: &Pubkey) -> Result<(Pubkey, u8)> {
+pub fn get_mint_owner_and_additional_data(
```

Comment:
Could you do me a favor and `pub(crate)` this while you're here?

Reply:
With pleasure